recipes/pyinstaller/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "PyInstaller" %}
+{% set version = "3.2" %}
+
+package:
+  name: {{ name.lower() }}
+  version: {{ version }}
+
+source:
+  fn: PyInstaller-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: ad924928983014e6b8ce5422d7687832
+  patches:
+    # Fixes an issue where libpython would not be found
+    # if it had a suffix (e.g. `m`). The PR used is
+    # linked below.
+    #
+    # https://github.com/pyinstaller/pyinstaller/pull/1971
+    #
+    - PR_1971.diff
+
+build:
+  number: 0
+  skip: true  # [win]
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+  entry_points:
+    - pyinstaller = PyInstaller.__main__:run
+    - pyi-archive_viewer = PyInstaller.utils.cliutils.archive_viewer:run
+    - pyi-bindepend = PyInstaller.utils.cliutils.bindepend:run
+    - pyi-grab_version = PyInstaller.utils.cliutils.grab_version:run
+    - pyi-makespec = PyInstaller.utils.cliutils.makespec:run
+    - pyi-set_version = PyInstaller.utils.cliutils.set_version:run
+  script:
+    - source activate "${CONDA_DEFAULT_ENV}"                    # [unix]
+    - export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"               # [linux]
```

Comment:
It won't find `libz` without this. Though the other libraries it found find. This was the workaround that seemed to solve the issue so we added it here.